documentation/mod-files.md

```diff
@@ -0,0 +1,124 @@
+# Module Files
+
+Module files hold information from a module that is necessary to compile 
+program units that depend on the module.
+
+## Name
+
+Module files must be searchable by module name. They are typically named
+`<modulename>.mod`. The advantage of using `.mod` is that it is consistent with
+other compilers so users will know what they are. Also, makefiles and scripts
+often use `rm *.mod` to clean up.
+
+The disadvantage of using the same name as other compilers is that it is not
+clear which compiler created a `.mod` file and files from multiple compilers
+cannot be in the same directory. This could be solved by adding something
```

Comment:
GCC implements -J, which tells the compiler where to put mod files.  -I sets up the search path for modules.  Directories specified in -J are also added to the search path.  PGI has -module, which adds to the search path; the docs didn't say that -module is also the destination directory for modules; I did not try it.

Reply:
I'll figure out what -module does and add some information about the options.